datastream-server/src/test/java/com/linkedin/datastream/server/assignment/TestPartitionAssignmentStrategyConfig.java

```diff
@@ -92,17 +75,17 @@ public void configValuesRevertedToEmptyWhenInvalidTest() {
 
     PartitionAssignmentStrategyConfig config = new PartitionAssignmentStrategyConfig(props);
     Assert.assertEquals(config.getMaxTasks(), Optional.empty());
-    Assert.assertEquals(config.getImbalanceThreshold(), Optional.empty());
-    Assert.assertEquals(config.getMaxPartitions(), Optional.empty());
-    Assert.assertEquals(config.getPartitionsPerTask(), Optional.empty());
+    Assert.assertEquals(config.getImbalanceThreshold(), (int) DEFAULT_IMBALANCE_THRESHOLD);
```

Comment:
such typecasting can be avoided if you set DEFAULT_IMBALANCE_THRESHOLD to be an int